webapps/camunda-commons-ui/lib/widgets/variables-table/cam-widget-variables-table.less

```diff
@@ -161,4 +161,26 @@
       margin-left: 0.25em;
     }
   }
+
+  .valid {
+    button {
+      padding: 0;
+    }
+    // Remove/fix .error-sign class when CAM-13580 is fixed
+    .error-sign {
+      display: block;
+      width:15px;
+      height:15px;
+      background-color: #d9534f;
+      -webkit-mask: url(../assets/images/error-sign.svg) no-repeat center;
+      mask: url(../assets/images/error-sign.svg) no-repeat center;
+    }
```

Comment:
It would also work to set the font-family back to the glyphicon font. As this is a workaround and also mentioned in the corresponding ticket, we can keep it like this until the root cause is investigated.

Reply:
If you agree, I prefer to keep it as a temporary workaround and fix it everywhere with CAM-13580. I don't think this workaround has a hard downside, and we know that it actually works and looks good. 🙂

Reply:
Yes, you can leave it! I was just thinking aloud :smile: